lib/easytest/matcher/contain_exactly.rb

```diff
@@ -2,7 +2,7 @@ module Easytest
   module Matcher
     class ContainExactly < Base
       def match?
-        (actual.to_a - expected).empty?
+        (Array(actual) - Array(expected)).empty?
```

Comment:
[note] [`Kernel#Array`](https://docs.ruby-lang.org/en/3.1/Kernel.html#method-i-Array) is more flexible.